plugin/trino-postgresql/pom.xml

```
@@ -81,6 +81,12 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
+            <!--
+            PostgreSQL connector does not work with newer drivers due to
+            fixes to handle of multidimensional arrays (>= 42.2.14) and
+            numeric types (>= 42.2.11) (Trino #7823)
```

Use a link that we can click instead. And maybe make this a TODO.

```suggestion
            numeric types (>= 42.2.11) (https://github.com/trinodb/trino/issues/7823)
```